engine/requirements.txt

```diff
@@ -7,7 +7,7 @@ phonenumbers==8.10.0
 celery[amqp,redis]==5.3.1
 redis==5.0.1
 humanize==0.5.1
-uwsgi==2.0.21
+uwsgi==2.0.23
```

Comment:
might need to check https://github.com/grafana/oncall-private/issues/2125

Reply:
this upgrade was performed in [v1.3.67](https://github.com/grafana/oncall/releases/tag/v1.3.67) 👍